budgetkey_data_pipelines/pipelines/entities/geocode_entities.py

```diff
@@ -16,21 +18,57 @@
                        "country": ["country"],
                        "po_box": ["pob", "zipcode", "postal_code", "pob_postal_code"],}
 
-
 class GeoCodeEntities(object):
 
+    PROVIDER_DEFAULT_PARAMS = {
+        "timeout": 5,
+        "sleep_seconds": 1
+    }
+
+    # the processor will try each provider that has the env_vars in order up to the limit
+    PROVIDERS = [
```

Comment:
We should add the OpenStreetMap geocoding provider, which has no hard limits (besides a 1 request per second maximum).